zerver/openapi/zulip.yaml

```diff
@@ -1050,6 +1069,7 @@ paths:
 
   /users:
     get:
+      operationId: get_members
```

Comment:
Let's use `get_users` here -- I want to rename the method in `python-zulip-api` to that.